superset/bin/superset

```diff
@@ -4,7 +4,82 @@ from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from superset.cli import manager
+import argparse
+import sys
+
+import gunicorn.app.base
+
+
+class GunicornSupersetApplication(gunicorn.app.base.BaseApplication):
+    def __init__(self, address, port, workers, timeout):
+        self.options = {
+            'bind': '%s:%s' % (address, port),
+            'workers': workers,
+            'timeout': timeout,
+            'limit-request-line': 0,
+            'limit-request-field_size': 0
+        }
+
+        super(GunicornSupersetApplication, self).__init__()
+
+    def load_config(self):
+        config = dict([(key, value) for key, value in self.options.iteritems()
+                       if key in self.cfg.settings and value is not None])
+        for key, value in config.iteritems():
+            self.cfg.set(key.lower(), value)
+
+    def load(self):
+        from superset import app
+
+        return app
+
+
+def run_server():
+    parser = argparse.ArgumentParser(description='Run gunicorn for superset')
+    subparsers = parser.add_subparsers()
+    gunicorn_parser = subparsers.add_parser('runserver')
+
+    gunicorn_parser.add_argument(
+        '-d', '--debug', action='store_true',
+        help='Start the web server in debug mode')
+    gunicorn_parser.add_argument(
+        '-a', '--address', type=str, default='127.0.0.1',
+        help='Specify the address to which to bind the web server')
+    gunicorn_parser.add_argument(
+        '-p', '--port', type=int, default=8088,
```

Comment:
This pull request has changed the default port number from `config.get("SUPERSET_WEBSERVER_PORT")` to `8088`. Effectively the `SUPERSET_WEBSERVER_ADDRESS`, `SUPERSET_WEBSERVER_PORT`, `SUPERSET_WORKERS`, and `SUPERSET_WEBSERVER_TIMEOUT` config variables have been removed. (They have no effect now.)

Is this intentional? These settings are still mentioned in the docs.